src/metabase/api/dashboard.clj

```diff
@@ -607,9 +608,7 @@
        true))
     (let [dashboard (t2/select-one :model/Dashboard id)]
       (events/publish-event! :event/dashboard-update {:object dashboard :user-id api/*current-user-id*})
-      (when update-dashcards-and-tabs?
-        ;; execute these events for old PUT /api/dashboard/:id/cards calls and new PUT /api/dashboard/:id calls, and not for old PUT /api/dashboard/:id calls
-        (track-dashcard-and-tab-events! dashboard @changes-stats))
+      (track-dashcard-and-tab-events! dashboard @changes-stats)
```

Comment:
This `when` was unnecessary, since `track-dashcard-and-tab-events!` should only track actual changes anyway